tests/unit/stats/test_measure_to_view_map.py

```diff
@@ -378,3 +378,20 @@ def test_export(self):
         measure_to_view_map._registered_measures = {}
         measure_to_view_map.export(view_data)
         self.assertTrue(True)
+
+    def test_export_duplicates_viewdata(self):
```

Comment:
This test might be too close to just asserting the implementation to be useful, you may want to change it to e.g. check that the end timestamp is getting set correctly. Since this PR is a bugfix I think it ought to include a test to prevent a regression, up to you whether you think this does that.

Reply:
Thanks for adding the test, I've fixed a failure with mock and made it test if `end_time` is properly updated.